Mathlib/Topology/Category/LightProfinite/Basic.lean

```diff
@@ -149,3 +159,15 @@ instance : EssentiallySmall LightProfinite.{u} where
   equiv_smallCategory := ⟨LightProfinite', inferInstance, ⟨LightProfinite.equivSmall⟩⟩
 
 end EssentiallySmall
+
+/--
+Many definitions involving universe inequalities in Mathlib are expressed through use of `max u v`.
```

Comment:
Do you think that [#3981](https://github.com/leanprover/lean4/pull/3981) can help here?

Reply:
I just merged master and tried changing `LightProfiniteMax.{u, w}` to `LightProfinite.{max u w}` and it didn't work. From the description of the PR you link it sounds like it should though, is mathlib not running on a version of Lean that has this feature?

Reply:
Indeed it seems that this feature is not in mathlib yet, I guess it will land with `v4.8.0-rc1`? Then we should be able to get rid of all these `TypeMax`, `TopCatMax`, etc. hacks?